pkg/features/kube_features.go

```diff
@@ -448,6 +448,18 @@ const (
 	// Requires the CRI implementation supports supplying the required stats.
 	PodAndContainerStatsFromCRI featuregate.Feature = "PodAndContainerStatsFromCRI"
 
+	// owner: @ahmedtd
+	// alpha: v1.33
+	//
+	// Enable PodCertificateRequest objects.
+	PodCertificateRequest featuregate.Feature = "PodCertificateRequest"
```

Comment:
I think we only need one gate for this one. There's no way to use this feature without the kubelet part.

Make the kube-apiserver startup confirm that `AuthorizeNodeWithSelectors` is enabled if `PodCertificateRequest` is enabled.  Fail to start otherwise.

Reply:
Done.